modules/apps/analytics/analytics-web/src/main/java/com/liferay/analytics/web/internal/servlet/taglib/AnalyticsTopHeadJSPDynamicInclude.java

```diff
@@ -237,6 +242,17 @@ private boolean _isConfigurationPortlet(
 		return false;
 	}
 
+	private boolean _isEditMode(HttpServletRequest httpServletRequest) {
+		String layoutMode = ParamUtil.getString(
+			httpServletRequest, "p_l_mode", Constants.VIEW);
+
+		if (!layoutMode.equals(Constants.VIEW)) {
```

Comment:
I think you can't assume that it's in EDIT mode when it's not VIEW. I'm pretty sure there are more modes, see https://liferayiseasy.blogspot.com/2015/09/portlet-modes-in-liferay.html